### DISCUSSION TITLE
翻译到了93%的时候被colab踢出来了……心碎

### DISCUSSION THREAD

**Discussion by pepperyoyo** [Ayanaminn/N46Whisper#44] [General]
感觉还是要一个本地版的翻译会比较好555

**Comment by Ayanaminn** [comment]
翻译部分可以先试试这个 https://github.com/gnehs/subtitle-translator-electron

**Reply by pepperyoyo** [reply]
谢谢，我试了一下，但这个翻译经常重复句子，感觉效果也不理想orz

**Comment by ShimamuraAdachii** [comment]
其实好像 被他踢出来没所谓 直接继续等他跑完 然后等他跳出来下载就行？
